.stylelintrc.json

```diff
@@ -2,5 +2,24 @@
   "extends": [
     "stylelint-config-standard-scss",
     "stylelint-config-prettier-scss"
-  ]
+  ],
+  "plugins": [
+    "stylelint-selector-bem-pattern"
+  ],
+  "rules": {
+    "plugin/selector-bem-pattern": {
+      "preset": "bem",
+      "implicitComponents": true,
+      "ignoreSelectors": [
+        "^html$",
+        "^body$"
+      ]
+    },
+    "selector-class-pattern": [
+      "^(?:(?:o|c|u|t|s|is|has|_|js|qa)-)?[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*(?:__[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:--[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:\\[.+\\])?$",
+      {
+        "message": "Class names should match the SUIT CSS naming convention"
+      }
+    ]
```

Comment:
Needed to ignore the default selector rule. Makes it compatible with BEM and SUIT
Source: https://github.com/btd1337/stylelint-config-sass-guidelines/blob/c1ba34cb6d549e2684d2ffc1154d407246623d7d/.stylelintrc#L12-L17